model/augmenters/base.py

```diff
@@ -5,17 +5,16 @@
 from utils.random import random_string
 from typing import Optional
 
+
 class Augmenter(Model):
     def __init__(self, id: Optional[str] = None):
         self.config = BaseConfig(force_fit=False)
-        self.id = (
-            self.__class__.__name__ + "-" + random_string(5) if id is not None else id
-        )
+        self.id = self.__class__.__name__ + "-" + random_string(5) if id is None else id
```

Comment:
Is your formatter set up differently?

Reply:
no, I needed tobremove is not, it was doing the opposite what we wanted before!